adapter/config/parser.go

```diff
@@ -77,15 +74,15 @@ const (
 // from where the executable is called from.
 //
 // Returns the configuration object that is initialized with default values. Changes to the default
-// configuration object is achieved through the configuration file. 
+// configuration object is achieved through the configuration file.
 func ReadConfigs() (*Config, error) {
 	onceConfigRead.Do(func() {
 		adapterConfig = defaultConfig
-		_, err := os.Stat(GetMgwHome() + relativeConfigPath)
+		_, err := os.Stat(pkgconf.GetMgwHome() + relativeConfigPath)
 		if err != nil {
 			logger.Fatal("Configuration file not found.", err)
 		}
-		content, readErr := ioutil.ReadFile(mgwHome + relativeConfigPath)
+		content, readErr := ioutil.ReadFile(pkgconf.GetMgwHome() + relativeConfigPath)
```

Comment:
can't we `config` as the package name instead of `pkgconf`

Reply:
this one's package name is config. the reference is pkgconf
```
pkgconf "github.com/wso2/product-microgateway/adapter/pkg/config"
```

so I would rename the reference as configCommon